README.md

```diff
@@ -19,39 +19,27 @@ The all-in-one starter kit for high-performance SaaS applications. With a few cl
 
 ![Architecture diagram](./public/architecture_diagram.svg)
 
-## Setup
+## Deploy with Vercel
+
+The Vercel deployment will guide you through creating a Supabase account and project as well as a Stripe test account and automatically creates the Stripe webhook endpoint for you (make sure to click the "Set up webhooks" button). After installation of the Supabase and Stripe integrations, all relevant environment variables will be set up so that the project is usable immediately after deployment 🚀
```

Comment:
Any reason not to automatically apply the webhooks instead of making them click a button?

Reply:
Not really, they could always update/delete it later. I've updated the integration to automatically set up the webhooks 👍

Reply:
Awesome! Thank you 🎉